compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -109,6 +111,8 @@ static void PrintMethodFields(
       // share the same request or response messages.
       p->Print(
```

Comment:
I meant the name, it use `Parser` because we did create `Parser` classes, now it should be `NanoFactory`.